permissions/pom-bom.yml

```diff
@@ -9,3 +9,4 @@ paths:
 - "io/jenkins/tools/bom/bom-2.138.x"
```

Comment:
I wonder if we should precreate the entry for 2.204.x to save time later.

Reply:
https://github.com/jenkins-infra/repository-permissions-updater/pull/1417